includes/admin/class-wcmp-export-consignments.php

```diff
@@ -180,6 +181,21 @@ public function getHsCode(WC_Product $product): int
         return (int) $hsCode;
     }
 
+    /**
+     * @param WC_Product $product
+     * @return string
+     */
+    public function getCountryOfOrigin(WC_Product $product): string
+    {
+        $defaultCountryOfOrigin = $this->getSetting(WCMP_Settings::SETTING_COUNTRY_OF_ORIGIN);
+        $productCountryOfOrigin = WCX_Product::get_meta($product, WCMP_Admin::META_COUNTRY_OF_ORIGIN, true);
+
+        $countryOfOrigin = ($defaultCountryOfOrigin ? $defaultCountryOfOrigin : $productCountryOfOrigin) ?
+            ($defaultCountryOfOrigin ? $defaultCountryOfOrigin : $productCountryOfOrigin) :
```

Comment:
Deze controle komt niet heel duidelijk over.
De shorthand notation is bedoeld voor een verkorte versie van een if/else en is bedoeld om het overzichtelijk mogelijk te houden en dat is het nu niet.

Eventueel kan je een aparte method maken waarin dan meerdere if statements te zien is.

Reply:
Done